Source/Xamarin/Prism.Forms/Navigation/IInternalNavigationParent.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Prism.Navigation
+{
+    /// <summary>
+    /// Interface for sharing NavigationParameters between a MultiPage and it's Children
+    /// </summary>
+    public interface IInternalNavigationParent
```

Comment:
I thought about something like having the NavigationService inject the parameters passed to the TabbedPage but I'm not sure that provides the right workflow or extensibility. Having the ```IInternalNavigationParent``` would allow for crazier scenarios where you might for some weird reason want the Parent to be able to change the parameters being sent to the children as the state of the application changes.